src/style.css

```
@@ -1,3 +1,76 @@
-body {
+html {
+  --blue: #00acdc;
+}
+
+body,
+ul,
+a {
   margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo-reference {
+  height: 40px;
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
```

```suggestion
}
```
just add the `padding: 0 50px` to the header